vagrant/provisioning/arkcase-all.yml

```diff
@@ -79,8 +79,10 @@
       tags: [core, pentaho, pentaho-configuration, marketplace]
     - role: snowbound
       tags: [core, snowbound, marketplace]
+      when: ecm_viewer is not defined
```

Comment:
This is nice, just need to remember to remove ecm_viewer: "pdftron" from the facts. Because, i can see it in more environments.

Reply:
but if it's defined, and set to pdftron, then these roles would be executed; it should be more like:

```yaml
when: (ecm_viewer | default('snowbound')) == 'snowbound'
```